init.mk

```diff
@@ -47,18 +47,42 @@ confirm:
 ########################################################################################################################
 # Core Dependencies
 GIT  ?= $(shell which git)
-DOCKER  ?= $(shell which docker)
+DOCKER ?= $(shell which docker)
+AWS-CLI ?= $(shell which aws)
```

Comment:
I would use an underscrore for `AWS_CLI` to keep it consistent with unix env-var naming